plugins/scheduled_task/batch/ScheduledTaskDryRunner/KScheduledTaskDryRunner.class.php

```diff
@@ -127,32 +127,39 @@ private function initRunData(KalturaBatchJob $job, KalturaScheduledTaskJobData $
 		$this->filter = $this->scheduledTaskProfile->objectFilter;
 	}
 
-	private function execDryRunInCSVMode($results, KalturaScheduledTaskJobData $jobData)
+	private function writeEntriesToCsv($entries)
+	{
+		$count =0;
+		foreach ($entries as $entry)
+		{
+			$csvEntryData = $this->getCsvData($entry);
+			fputcsv($this->handle, $csvEntryData, ",");
+			$count++;
```

Comment:
$count is always count ($entries) why do we need to count it here ?
This function does not need to return anything